pos_order_to_sale_order/README.rst

```diff
@@ -15,74 +15,87 @@ creation from the Point of Sale.
 
 
-Three options are available:
+Four options are available:
 
-#. '**Create a draft Order**'
-  A new sale order in a draft mode will be created that can be changed later.
+#. '**Allow PoS order**'
 
-.. figure:: static/description/pos_create_picking_option_1.png
-   :width: 800 px
+   A PoS Order is the default type of order (opposed to sale orders).
+   If you want to use only sales order, uncheck this option.
 
-#. '**Create a Confirmed Order**'
-  A new sale order will be created and confirmed.
+#. '**Allow draft order**'
 
-.. figure:: static/description/pos_create_picking_option_2.png
-   :width: 800 px
+   A draft sale order is a quotation. 
+   It can be changed and confirmed later from the backend.
+   No payment can be taken with draft orders.
 
-#. '**Create Delivered Picking**' (by default)
-  A new sale order will be created and confirmed. the associated picking
-  will be marked as delivered.
+#. '**Allow confirmed order**'
+
+   A confirmed sale order without picking.
+   Useful if you don't manage stock or sell services otherwise
+   use 'delivered picking'.
+
+#. '**Allow delivered picking**'
+
+   A sale order will be created and confirmed. The associated picking
+   will be marked as delivered.
+   Use this option if you delivery your customer frequently and bill him periodically
+
+If multiple options are activated, buttons will appear on the payment screen.
 
-.. figure:: static/description/pos_create_picking_option_3.png
-   :width: 800 px
 
 Configuration
 =============
 
 To configure this module, you need to:
 
 #. Go to Point Of Sale / Configuration / Point of Sale
-#. Check the box 'Create Sale Orders'
-#. Select the desired default behaviour
+#. Select the desired default behavior
 
-.. figure:: static/description/pos_config_form.png
-   :width: 800 px
 
 Usage
 =====
 
+In the PoS, on the payment screen, new buttons will appear according
+ to options set in the configuration.
+
+.. figure:: static/description/pos_order_to_sale_order.png
+   :width: 800 px
+
+
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/184/8.0
 
 Technical Notes
 ===============
 
-* Some hooks are defined in the JS file, to define custom behaviour after
+* Some hooks are defined in the JS file, to define custom behavior after
   having created the sale order (and the stock picking).
 
 * Some prepare functions are available in the sale.order model, to overload
   the creation of the sale order.
 
-* You could be interested by another module, pos_sale_order, that completely
-  alter Point of Sale module, avoiding creating Pos Orders, and creating
-  allways Sale Orders.
-  This module is a WIP state, and is available here:
-  https://github.com/OCA/pos/pull/35
+* An additional, not displayed option, is to allow payment for sale orders.
+  It requires an additinnal module (not yet written) to handle it correctly.
```

Comment:
Why to include this yet then?

Reply:
To be clear that this modules doesn't handle payments.

What do you suggest then ?